newportApp/src/AG_CONEX.cpp

```diff
@@ -496,12 +519,29 @@ asynStatus AG_CONEXAxis::poll(bool *moving)
   sprintf(pC_->outString_, "%dTS", pC_->controllerID_);
   comStatus = pC_->writeReadController();
   if (comStatus) goto skip;
-  // The response string is of the form "1TSabcdef"
-  count = sscanf(pC_->inString_, "%*dTS%*4c%x", &status);
-  if (count != 1) goto skip;
+  
+  if (conexModel_ == ModelDL) {
+	  // The TS command returns 8 characters (1TSabcdefgh).
+	  // The first character (a) represents the status bits in Hexadecimal.
+	  // The next 5 characters (bcdef) represent the error bits in Hexadecimal.
+	  // The last two characters (gh) represent the Controller state in Hexadecimal.
+	  // we read last two hex characters, the state
+	  count = sscanf(pC_->inString_, "%*dTS%*6c%x", &status);
+	  if (count != 1) goto skip;
+  } 
+  else {
```

Comment:
This formatting (newline preceding `else`) does not appear to be consistent with what the rest of the code uses.

Yes, there is one place earlier in the code that uses the same style, so there already is some inconsistency in the exsting code, but the prevailing style seems to be no newline before `else`, so I suggest going with that one.